comments/views.py

```diff
@@ -94,3 +106,62 @@ def post(self, request):
 
         referer_url = urlunparse(parsed_url)
         return redirect(to=referer_url)
+
+
+class CommentsModerationView(ListView):
+    model = XtdComment
+    template_name = 'comments/moderation.html'
+    context_object_name = 'comments'
+    paginate_by = 10
+
+    @method_decorator(login_required)
+    def dispatch(self, request, *args, **kwargs):
+        if not (request.user.has_perm('django_comments_xtd.can_moderate') or settings.ENABLE_FE_COMMENTS_MODERATION):
+            raise PermissionDenied(
+                "You do not have permission."
+            )
+        return super().dispatch(request, *args, **kwargs)
+
+    def get_queryset(self):
```

Comment:
No need to change anything, but I do feel there ought to be an automated test for this. This method is ripe for refactoring as well.